src/lib/learn-client/index.ts

```diff
@@ -1,6 +1,6 @@
 function getFetch() {
   // Note: purposely doing a conditional require here so that `@vercel/fetch` is not included in the client bundle
-  if (typeof window === 'undefined') {
+  if (typeof window === 'undefined' || typeof window.fetch !== 'function') {
```

Comment:
Perhaps you want to mock that data call in the test? [I used nock ](https://github.com/hashicorp/dev-portal/blob/main/src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts#L91-L95)to handle this for the tutorial link rewrites work.

Reply:
From what I see it looks like the test would actually fetch from the API currently? Which may not be what we want.

Reply:
Ah okay that makes sense - I also didn't realize https://github.com/hashicorp/dev-portal/blob/main/src/lib/remark-plugins/rewrite-tutorial-links/index.ts existed, seems like I might be able to re-use parts of that...

eg as a _very_ skateboard version we could construct a `Link` mdast node and make a named export of `handleRewriteTutorialsLink`; which would avoid doing any refactoring at all; maybe as a step beyond that we could split out a function that acts on `url` rather than `node.url` 🤔 

Apologies for the thrash here, I should have looked around a bit first 🙇‍♂️

Reply:
✅ Update here: have reverted this and other changes that were previously in place to allow the tests for `detect-and-reformat-learn-url` to fetch from the Learn API. Now using `nock` instead 👍